OJP/OJP_Trips.xsd

```diff
@@ -265,7 +265,7 @@
 			<xs:documentation>Structure for trip overview information.</xs:documentation>
 		</xs:annotation>
 		<xs:sequence>
-			<xs:element name="TripId" type="xs:NMTOKEN">
+			<xs:element name="TripId" type="ObjectIdType">
```

Comment:
Is it on purpose this is still TripId and not Id?

Reply:
will change